tests/test.py

```diff
@@ -16,7 +18,12 @@ def test_node_reg(self):
             }
         }
 
-        re = self._post('/v1/node_register', payload)
+        signature, public_address = helpers.sign_message_with_static_key('')
```

Comment:
Since `public_address` is not used, we can reduce local variables by omiting it:
```py
signature, _ = ...
```
This applies for most of `sign_message_with_static_key` usages.

Reply:
done